src/rgw/rgw_object_expirer_core.h

```diff
@@ -4,12 +4,6 @@
 #ifndef CEPH_OBJEXP_H
 #define CEPH_OBJEXP_H
 
-#include <errno.h>
```

Comment:
@mikulely We don't seem to be overly consistent about it. I've occasionally been moving standard headers together and after "our" headers, since this is one way to help ensure that our headers are including what they need. OTOH, it's true that if we aren't really following through on that the benefit isn't entirely palpable. If you feel strongly about it, I can move them back, but I noticed that taking out our atomic header is causing issues like needing to rewrite dout.